### PR TITLE
[NEW UI] Feat : add connection to page post update

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Then update is ended with success we loading the post update page.
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Sponsor company   | @PrestaShopCorp
| How to test?      | Try and update, if the update finish without blocking error you need to be redirected to the post update page.

### CODE REVIEW

**Review by tblivet — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
I think we need a rebase here

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by tblivet — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
QA by Dave

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
